app/javascript/controllers/media_tag_controller.js

```diff
@@ -73,6 +73,22 @@ export default class extends Controller {
           const event = new CustomEvent('time-update', { detail: timestamp })
           window.dispatchEvent(event)
         })
+       
+        // When at least one of the tracks has been loaded, trigger transcript loading
+        // Before the tracks are loaded, the cues will not be recognized and the transcript will
+        // not load properly
+        player.textTracks().on('addtrack', () => {
```

Comment:
when videojs starts to parse/add tracks, we can check if the tracks exist, and if so, will trigger the "media-data-loaded" event